File: java23/SecondLastElement.java
//Problem 2

package java23;
import java.util.List;

public class SecondLastElement {
    public static void main(String[] args) {
        List<String> list = List.of("a", "b", "c", "d", "e");

        if (list.isEmpty()) {
            System.out.println("Empty List");
        } else {
            String secondlastElement = list.get(list.size() - 2);
            System.out.println("Second Last Element : " + secondlastElement);
        }
    }
    
}

File: python23/second_last_element.py
#Problem 2

def find_element(one):
    if len(one) < 2:
        return None
    
    return one[-2]
    

example_list = ['a', 'b', 'c', 'd']
second_last_element = find_element(example_list)

print("Second Last Element : ", second_last_element)

